### PR TITLE
Feature/user management

### PR DESCRIPTION
**Pull request by diwakar-s-maurya**
- display number of contributions done by a user

### CODE REVIEW

File: src/app/action/action.html [hunk 1]
```diff
@@ -21,50 +21,20 @@
...
-            <span type="button" class="btn btn-default btn-xs glyphicon glyphicon-thumbs-up pull-right"></span>
+      <li data-ng-repeat="item in action.items.slice().reverse()">
+        <article class="sticky">
+          <div class="sticky__head">
+            <select ng-options="asn for asn in action.assignee" ng-model="item.name" ng-change="action.modify(item)" class="selectpicker">
+            </select>
             <button class="btn btn-default btn-xs pull-right btn-danger glyphicon glyphicon-trash" ng-click="action.remove($index)"></button>
           </div>
-          <div class="panel-body fixed-panel-body" style="max-height: 10; min-height: 10; overflow-y: scroll;">
+          <div class="sticky__body" style="max-height: 10; min-height: 10; overflow-y: scroll;">
```
Comment (skeep) [inline src/app/action/action.html]: no inline styling plz

File: src/app/action/action.html [hunk 2]
```diff
@@ -21,50 +21,20 @@
...
-        <article class="panel panel-default">
-          <div class="panel-heading fixed-panel-head">
-            {{ item.name }}
-            <span type="button" class="btn btn-default btn-xs glyphicon glyphicon-thumbs-up pull-right"></span>
+      <li data-ng-repeat="item in action.items.slice().reverse()">
+        <article class="sticky">
+          <div class="sticky__head">
+            <select ng-options="asn for asn in action.assignee" ng-model="item.name" ng-change="action.modify(item)" class="selectpicker">
+            </select>
             <button class="btn btn-default btn-xs pull-right btn-danger glyphicon glyphicon-trash" ng-click="action.remove($index)"></button>
```
Comment (skeep) [inline src/app/action/action.html]: why the icon class is inside button. plz use separate tag like `<i>`

**Comment by skeep** [comment]
BTW why this PR and @vatsalya25 PR is almost same?

**Comment by diwakar-s-maurya** [comment]
Qu: BTW why this PR and @vatsalya25 PR is almost same?
Ans: because i had pulled code from vatsalya's branch.
I am closing this pull request. I will create PR when his branch gets merged to develop

**Pull request closed without merging**